### DISCUSSION TITLE
VMess 关闭加密，专门中转加密的 WireGuard ，这样做是否靠谱？

### DISCUSSION THREAD

**Discussion by wy16W2pIilK1xgqN** [XTLS/Xray-core#1756] [Q&A]
需要使用 WireGuard，但运营商封锁了该协议，目前使用 VMess 中转 WireGuard。
但这种方法加密了两次，打算关闭 VMess 的加密，请问这样做靠谱吗？

**Comment by wangmice** [comment]
哪里的运营商封锁了wireguard协议？只要不出墙应该用的好好的吧

**Reply by wy16W2pIilK1xgqN** [reply]
出不了城域网，今年1月份开始的，应该是本地运营商策略配置错误。

**Comment by wangmice** [comment]
是不是触发udp qos了？

**Reply by wy16W2pIilK1xgqN** [reply]
不是，哪怕只发Ping包，5分钟内必然阻断端口。
当然可以换端口，但不停搞太麻烦了。.

**Comment by icepigeon-dev** [comment]
可以用VLESS，已经用了一年多了，一切正常
